### PR TITLE
Enable online-analysis by default

### PR DESCRIPTION
**Pull request by IAlibay**
It'll write out a yaml file every 200 MCMC steps (so by default 200 ps).

I set the target error to 0.0 that way there shouldn't be any early termination by default.

### CODE REVIEW

File: openfe/tests/protocols/test_openmm_equil_rfe_protocols.py
Comment (IAlibay) [inline openfe/tests/protocols/test_openmm_equil_rfe_protocols.py:814]: re-enable before merging.

**Review by dwhswenson — APPROVED** [review]

**Comment by IAlibay** [comment]
blergh keys.. always keys

**Comment by IAlibay** [comment]
@richardjgowers I don't know if you were trying to hold off on merging things for the v0.8 release, so I'll wait on you before merging.

File: openfe/protocols/openmm_rfe/equil_rfe_settings.py
Comment (richardjgowers) [inline openfe/protocols/openmm_rfe/equil_rfe_settings.py:180]: are these two settings that every 200 steps you check if the target error is below 0.0?  There's a `unit.timestep` for denoting steps if that's the case, else it's `unit.picosecond` maybe?

Comment (richardjgowers) [reply openfe/protocols/openmm_rfe/equil_rfe_settings.py:180]: Or more generally `Interval at which` is ambiguous and could mean either simulated time or timesteps so let's rephrase

File: openfe/protocols/openmm_rfe/equil_rfe_settings.py
Comment (richardjgowers) [inline openfe/protocols/openmm_rfe/equil_rfe_settings.py:185]: If we can, suggesting a sensible default for when people do want to use this feature would be helpful.  Was 0.1 a reasonable value when using this feature?

File: openfe/protocols/openmm_rfe/equil_rfe_settings.py
Comment (IAlibay) [reply openfe/protocols/openmm_rfe/equil_rfe_settings.py:172]: how about this?

**Comment by IAlibay** [comment]
Going ahead and merging so that it doesn't hold back tutorials etc...

**Pull request merged**
